scripts/project-setup-scripts/google-project-viewer.yaml

```diff
@@ -0,0 +1,26 @@
+title: "google_project_viewer"
```

Comment:
`terra_workspace_writer`

Reply:
after looking at this again, I'm wondering if "terra_workspace_can_compute" might be most accurate. Not all writers have can-compute

Reply:
I went ahead and made the change but can revert it if we decide to